### DISCUSSION TITLE
Two TV Boxes - With same MAC address causing issue

### DISCUSSION THREAD

**Discussion by dseknat10** [devmfc/debian-on-amlogic#116] [Q&A]
I have recently installed the Debian on two tv boxes and they are working fine.
But when I connect to the network, they are getting the same ip address, because they use same MAC address. 
I even tried with Macchanger, but still having same ip address contention.
It seems like to the permanent Mac address remains the same.

Could you please let me know how to change the Mac address?

**Comment by devmfc** [comment]
You could try to add the line with a mac address in _/boot/boot.config_ like:
`ethaddr=02:00:00:1d:04:01`

Did you flash the same firmware tot these boxes? Or did you install them with the same USB Flash drive /SDCARD?

**Reply by volantinh0** [reply]
Hi, I'm facing the same problem with the "shared" mac between two s905x
Initially, I had installed the same distro with the same memory card
Later, I tried Debian on one and Ubuntu on the other, and the problem continues
Lastly, I tried adding this line to /boot/boot.config, but I still have the same problem

**Reply by volantinh0** [reply]
Edit: 
I managed to install Ubuntu on one and Debian on the other, and now both works
